src/context.rs

```diff
@@ -0,0 +1,43 @@
+use crate::Actor;
+use std::marker::PhantomData;
+
+/// `Context` is used to signal things to the [`ActorManager`](struct.ActorManager.html)'s
+/// management loop. It can be used to stop the actor ([`Context::stop`](struct.Context.html#method.stop)) or
+/// queue a message to be processed by the actor immediately after it has finished processing the
+/// current message ([`Context::notify`](struct.Context.html#method.notify)).
+pub struct Context<A: Actor + ?Sized> {
+    pub(crate) running: bool,
+    phantom: PhantomData<A>, // TODO(weak_address)
```

Comment:
For the TODO and commented out notifications, do you want to handle that in this PR?

Reply:
No. Weak_address will be the next branch/pr name

Reply:
What about notifications?

Reply:
Same thing ;P (weak address allows for notifications. it will be in the same pr)